database/seeders/SharedEnvironmentVariableSeeder.php

```
@@ -32,5 +33,29 @@ public function run(): void
             'project_id' => 1,
             'team_id' => 0,
         ]);
+
+        // Add predefined server variables to all existing servers
+        $servers = \App\Models\Server::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Server' in method 'run'. (undefined)

(StaticAccess)